description.ext

```diff
@@ -3,27 +3,41 @@
 	Missionsnamen. Zudem dient sie als erweiterter Arm der A3-Konfiguration,
 	sodass hier auch neue Sounds, Menüs etc. definiert werden können.
 */
-/* Beschreibungen, Ladebilder etc. */
-author = "DeinName";
-briefingName = "RR CO20 Beispielmissionsname 1.0";
-onLoadName = "MissionsÜberschrift";
-loadScreen = "";
-onLoadMission = "KurzeBeschreibungWährendDesLadensDerMission";
-overviewText = "KurzeBeschreibungFürÜbersichtImServerBeimAuswählenDerMission";
-gameType = COOP; // (https://community.bistudio.com/wiki/Multiplayer_Game_Types)
+/* Beschreibungen, Ladebilder etc. Diese sollten alle ersetzte werden! */
+Author = "DeinName";
+IntelBriefingName = "RR CO20 Beispielmission v1"
```

Comment:
The naming guidelines for mission versioning should be the same, as for the mission file.
Thus 1v0 would be a better example.

Reply:
Will fix